### ISSUE TITLE
[PRE REVIEW]: ReactiveMP.jl: A Julia Package for Reactive Message Passing-based Bayesian Inference

### ISSUE THREAD

**Issue by whedon** [JuliaCon/proceedings-review#84]
**Submitting author:** @bvdmitri (<a href="http://orcid.org/0000-0001-9655-7986">Dmitry Bagaev</a>)
**Repository:** <a href="https://github.com/biaslab/ReactiveMP.jl" target ="_blank">https://github.com/biaslab/ReactiveMP.jl</a>
**Version:** 
**Editor:** Pending
**Reviewer:** Pending
**Managing EiC:** Valentin Churavy

**Author instructions**

Thanks for submitting your paper to JCON @bvdmitri. **Currently, there isn't an JCON editor assigned** to your paper.

The author's suggestion for the handling editor is @matbesancon.

@bvdmitri if you have any suggestions for potential reviewers then please mention them here in this thread (without tagging them with an @). In addition, [this list of people](https://bit.ly/jcon-reviewers) have already agreed to review for JCON and may be suitable for this submission (please start at the bottom of the list).

**Editor instructions**

The JCON submission bot @whedon is here to help you find and assign reviewers and start the main review. To find out what @whedon can do for you type:

```
@whedon commands
```

**Comment by whedon** [comment]
Hello human, I'm @whedon, a robot that can help you with some common editorial tasks.

**:warning: JOSS reduced service mode :warning:**

Due to the challenges of the COVID-19 pandemic, JOSS is currently operating in a "reduced service mode". You can read more about what that means in [our blog post](https://blog.joss.theoj.org/2020/05/reopening-joss).

For a list of things I can do to help you, just type:

```
@whedon commands
```

For example, to regenerate the paper pdf after making changes in the paper's md or bib files, type:

```
@whedon generate pdf
```

**Comment by whedon** [comment]
```
Software report (experimental):

github.com/AlDanial/cloc v 1.88  T=0.43 s (718.2 files/s, 133827.1 lines/s)
-------------------------------------------------------------------------------
Language                     files          blank        comment           code
-------------------------------------------------------------------------------
Julia                          257           3986            696          11806
SVG                              5              2              0           3392
Jupyter Notebook                29              0          31807           2946
TOML                             4            266              1           1256
Markdown                         9            293              0            636
YAML                             3              0              0            115
-------------------------------------------------------------------------------
SUM:                           307           4547          32504          20151
-------------------------------------------------------------------------------


Statistical information for the repository 'a7db76959da12d92d2502804' was
gathered on 2021/09/23.
The following historical commit information, by author, was found:

Author                     Commits    Insertions      Deletions    % of changes
Dmitri Bagaev                    2           552            552          100.00

Below are the number of rows from each author that have survived and are still
intact in the current revision:

Author                     Rows      Stability          Age       % in comments
```

**Comment by whedon** [comment]
PDF failed to compile for issue #84 with the following error: 
```
 Can't find any papers to compile :-(

```

**Comment by bvdmitri** [comment]
Paper is located in the different branch: https://github.com/biaslab/ReactiveMP.jl/tree/paper/julia_proceedings_2021

**Comment by matbesancon** [comment]
@whedon assign @matbesancon as editor

**Comment by whedon** [comment]
OK, the editor is @matbesancon

**Comment by matbesancon** [comment]
@whedon build pdf from branch paper

**Comment by whedon** [comment]
I'm sorry human, I don't understand that. You can see what commands I support by typing:

```
@whedon commands
```

**Comment by matbesancon** [comment]
@whedon compile pdf from branch paper/julia_proceedings_2021

**Comment by whedon** [comment]
I'm sorry human, I don't understand that. You can see what commands I support by typing:

```
@whedon commands
```

**Comment by matbesancon** [comment]
@whedon generate pdf from branch paper/julia_proceedings_2021

**Comment by whedon** [comment]
```
Attempting PDF compilation from custom branch paper/julia_proceedings_2021. Reticulating splines etc...
```

**Comment by whedon** [comment]
PDF failed to compile for issue #84 with the following error: 
```
 Latexmk: This is Latexmk, John Collins, 17 Jan. 2018, version: 4.55.
Rule 'pdflatex': Rules & subrules not known to be previously run:
   pdflatex
Rule 'pdflatex': The following rules & subrules became out-of-date:
      'pdflatex'
------------
Run number 1 of rule 'pdflatex'
------------
------------
Running 'pdflatex  -recorder  "paper.tex"'
------------
Rule 'pdflatex': File changes, etc:
   Changed files, or newly in use since previous run(s):
      'paper.aux'
------------
Run number 2 of rule 'pdflatex'
------------
------------
Running 'pdflatex  -recorder  "paper.tex"'
------------
Rule 'pdflatex': File changes, etc:
   Changed files, or newly in use since previous run(s):
      'paper.out'
------------
Run number 3 of rule 'pdflatex'
------------
------------
Running 'pdflatex  -recorder  "paper.tex"'
------------
Failure to make 'paper.pdf'
Collected error summary (may duplicate other messages):
  pdflatex: Command for 'pdflatex' gave return code 1
      Refer to 'paper.log' for details
Looks like we failed to compile the PDF

```

**Comment by matbesancon** [comment]
hi @bvdmitri, could you adapt you submission to the template:
https://github.com/JuliaCon/JuliaConSubmission.jl/
there are some important files in the folder that are not in yours, in particular the .latexmkrc file

**Comment by bvdmitri** [comment]
Dear @matbesancon . Sorry for the inconvenience, I missed that file in the original repository for some reason. I added it now and also adjusted text a bit so it fits in one page with the new header. Should be fine now.

**Comment by matbesancon** [comment]
@whedon generate pdf from branch paper/julia_proceedings_2021

**Comment by whedon** [comment]
```
Attempting PDF compilation from custom branch paper/julia_proceedings_2021. Reticulating splines etc...
```

**Comment by whedon** [comment]
:point_right::page_facing_up: [Download article proof](https://raw.githubusercontent.com/JuliaCon/proceedings-papers/jcon.00084/jcon.00084/10.21105.jcon.00084.pdf) :page_facing_up: [View article proof on GitHub](https://github.com/JuliaCon/proceedings-papers/blob/jcon.00084/jcon.00084/10.21105.jcon.00084.pdf) :page_facing_up: :point_left:

**Comment by matbesancon** [comment]
perfect it compiles. General remark, the abstract is a bit short and would benefit at least from additional definitions, context, and a section specifically on the package itself

**Comment by bvdmitri** [comment]
Dear @matbesancon, I will ask just to be sure. You suggest to change abstract only?

**Comment by matbesancon** [comment]
these comments are for the whole paper. You submitted it in a short version (extended abstract) which are one page + references, so you still have some place to extend a bit

**Comment by bvdmitri** [comment]
@matbesancon I thought references have to be on the same page, If I extend my text references go on the page number 2. Is that allowed? If yes, then I would be happy to extend it.

**Comment by matbesancon** [comment]
yes that's allowed, see https://juliacon.github.io/proceedings-guide/author/
> one page + references

so the references are excluded from the page limit

**Comment by bvdmitri** [comment]
Oh, I saw that but misread the explanation. My bad, I will change my proposal today or tomorrow and return to you, thanks!

**Comment by bvdmitri** [comment]
Dear @matbesancon, new version of my proposal is available on the same branch.

**Comment by matbesancon** [comment]
@whedon generate pdf from branch paper/julia_proceedings_2021

**Comment by whedon** [comment]
```
Attempting PDF compilation from custom branch paper/julia_proceedings_2021. Reticulating splines etc...
```

**Comment by whedon** [comment]
:point_right::page_facing_up: [Download article proof](https://raw.githubusercontent.com/JuliaCon/proceedings-papers/jcon.00084/jcon.00084/10.21105.jcon.00084.pdf) :page_facing_up: [View article proof on GitHub](https://github.com/JuliaCon/proceedings-papers/blob/jcon.00084/jcon.00084/10.21105.jcon.00084.pdf) :page_facing_up: :point_left:

**Comment by matbesancon** [comment]
thank you, we will now find reviewers for this paper

**Comment by matbesancon** [comment]
@whedon start review

**Comment by whedon** [comment]
OK, I've started the review over in https://github.com/JuliaCon/proceedings-review/issues/91.
